xtask/src/build_docker_image.rs

```diff
@@ -236,6 +236,9 @@ pub fn build_docker_image(
         for arg in &build_arg {
             docker_build.args(&["--build-arg", arg]);
         }
+        if let Ok(env_verbose) = std::env::var("VERBOSE") {
+            docker_build.args(&["--build-arg", &format!("VERBOSE={env_verbose}")]);
+        }
```

Comment:
what about making verbose an `u8`, and if verbosity > 1 then set build-arg `VERBOSE`

```
#[clap(action = clap::ArgAction::Count)]
verbose: u8
```

Reply:
Sure that works, maybe we should have something like `-vv` or similar then instead?

Reply:
isn't that what it enables?